packages/x-date-pickers/src/internals/components/PickersArrowSwitcher/PickersArrowSwitcher.types.tsx

```diff
@@ -39,9 +39,9 @@ export interface PickersArrowSwitcherProps
 
 export type PickersArrowSwitcherOwnerState = PickersArrowSwitcherProps;
 
-export interface PickersArrowSwitcherComponentsPropsOverrides {}
+export interface PickersArrowSwitcherSlotPropsOverrides {}
```

Comment:
There is still the similar `PickersCalendarHeaderComponentsPropsOverrides` leftover 🙈